cypress/component/CommentCard.cy.jsx

```diff
@@ -0,0 +1,75 @@
+import CommentCard from "../../src/components/TypesOfRecipeCards/CommentCard";
+import { testUser } from "../testData/testUser";
+
+const createTestUser = (isRecommend) => ({
+  ...testUser,
+  reviews: [
+    {
+      ...testUser.reviews[0],
+      isRecommend,
+    },
+  ],
+});
+
+const mountCommentCard = (user, review) => {
+  cy.mount(<CommentCard reviewer={user} review={review} />);
+};
+
+describe("CommentCard", () => {
+  context("when user review is empty", () => {
+    const user = { ...testUser, reviews: [] };
+
+    beforeEach(() => {
+      mountCommentCard(user, null);
+    });
+
+    it("CommentCard does not render", () => {
+      cy.get("comment-card-details").should("not.exist");
+    });
+  });
+
+  context("when user review is not empty", () => {
+    const userNotRecommends = createTestUser(false);
```

Comment:
I still think you should just create a user and change the `isRecommended` within the test. I find the variable names a bit confusing and the `createTestUser` helper is more complicated than needed.